lib/src/logger.dart

```diff
@@ -212,56 +212,56 @@ class Logger {
   ///
   /// See [log] for information on how non-String [message] arguments are
   /// handeled.
-  void finest(message, [Object? error, StackTrace? stackTrace]) =>
+  void finest(Object? message, [Object? error, StackTrace? stackTrace]) =>
```

Comment:
I'm unsure of this one. Do we know that folks are expecting to be able to log `null`?

On the other hand, if making it non-nullable means a bunch of people end up naively doing `!` and having otherwise irrelevant logging crash the app, maybe it's better this way.

Reply:
Right, I want people to be able to easily log _potentially_ nullable things and not accidentally throw.